pom.xml

```diff
@@ -107,6 +107,31 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-resteasy-reactive</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkiverse.web-bundler</groupId>
+      <artifactId>quarkus-web-bundler</artifactId>
+      <version>${version.quarkus-web-bundler}</version>
+    </dependency>
+    <!-- Web Dependencies -->
+    <dependency>
+      <groupId>org.mvnpm</groupId>
+      <artifactId>lit</artifactId>
+      <version>3.1.2</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mvnpm</groupId>
+      <artifactId>lodash</artifactId>
+      <version>4.17.21</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mvnpm.at.types</groupId>
+      <artifactId>lodash</artifactId>
+      <version>4.14.202</version>
+      <scope>provided</scope>
+      <classifier>sources</classifier>
+    </dependency>
```

Comment:
for other dependencies, we've tried to keep the versions under the dependency management section, and here, just use them without versions.